### PR TITLE
Fix navigation JS error

### LINKED ISSUE #3654: [BUG] mobile view: top button with JavaScript error
**Describe the bug**
in mobile view
no article is selected

click on the top button in the navigation
image

an JavaScript error expires
image

**Environment information (please complete the following information):**

 - Browser: Firefox 88
 - FreshRSS version: 1.18.0

### PR DESCRIPTION
**Pull request by aledeg**
Closes #3654

Changes proposed in this pull request:

- Fix navigation error when no article is selected

How to test the feature manually:

1. Display articles without selecting one
2. Use the up navigation button
3. Check that it does not generate an error in the console

Pull request checklist:

- [x] clear commit messages
- [x] code manually tested
- [ ] unit tests written (optional if too hard)
- [ ] documentation updated

[Additional information can be found in the documentation](https://github.com/FreshRSS/FreshRSS/tree/edge/docs/en/developers/04_Pull_requests.md).

Before, the UP navigation button was generating an error when no article was
selected. The process to navigate to the first element was using the position
of the current element to compute the offset, thus generating said error.
Now, if no article is selected, the first article is used as a fall-back.

### CODE REVIEW

**Review by Frenzie — APPROVED** [review]
Not too sure about the variable name with this change since it's not an item with all you might expect from that, but it's limited in scope to the line right below so I suppose it's fine.

**Comment by aledeg** [comment]
@Frenzie I've checked the behavior for the other navigation button when nothing is selected. It works by selecting the first element of the list. I guess it can be safe to assume that the active element is the first one when none are selected.

**Comment by Frenzie** [comment]
@aledeg But the first element is an actual item.

**Comment by aledeg** [comment]
I am not following you. Could you rephrase so I can understand what you have in mind?

**Comment by Frenzie** [comment]
My bad, .flux is a bit misleading. That is an item, not the container of the items.

**Comment by aledeg** [comment]
Oh! I see what you meant now. I understand your worries.
I agree with you that `.flux` could be misleading.
